auth.go

```diff
@@ -137,12 +137,27 @@ func ProvideAuthChain(in AuthChainIn) (AuthChainOut, error) {
 			return AuthChainOut{Chain: &alice.Chain{}}, emperror.With(err, "failed to create resolver")
 		}
 
-		options = append(options, basculehttp.WithTokenFactory("Bearer", basculehttp.BearerTokenFactory{
-			DefaultKeyId: DefaultKeyID,
-			Resolver:     resolver,
-			Parser:       bascule.DefaultJWTParser,
-			Leeway:       jwtVal.Leeway,
-		}))
```

Comment:
Oops, just realized: I think we need this code in the `if` part of the `if/else`, right? So if super users aren't enabled, we still accept JWTs.

Reply:
We still have it in line 153